dali/operators/decoder/nvjpeg/permute_layout.cu

```diff
@@ -100,6 +100,14 @@ void PlanarRGBToGray(Output *output, const Input *input, int64_t npixels,
   planar_rgb_to_gray<<<num_blocks, block_size, 0, stream>>>(output, input, npixels);
 }
 
+template <typename Output, typename Input>
```

Comment:
Note: Adding this wrapper here because calling the kernel directly breaks the build for *.cc compilation units that include the nvjpeg decoupled API header.